examples/config_dc2_test_med.yaml

```diff
@@ -0,0 +1,85 @@
+# This is an example of the campaign level template specification file for dc2-test-med type campaigns
+
+# Import the Step templates for DC2
+- import: "${CM_CONFIGS}/config_dc2_steps.yaml"
+# Now build the Campaign Template for DC2 test med
+- SpecBlock:
+      name: d2c_campaign
+      # Define collections associated with the campaign
+      collections:
+          root: 'cm/dc2_test_med'
+      # Define the steps, their connections and override parameters as needed
+      child_config:
+          step1:
```

Comment:
reference, it could be 
      - step:
            spec_block: step1
            otherstuff: stuff
     - step: 
            spec_block: step2
             ...